packages/tools/pie-icons-webc/helpers/index.js

```diff
@@ -0,0 +1,5 @@
+const removeHyphenBeforeDigits = (name) => name.replace(/\-(\d+)/, '$1'); // eslint-disable-line no-useless-escape

```

Comment:
Doesn't need to be part of this PR, but would be good to tidy this up so that both the vue and webc packages just use the same function for this (if that's possible using the `pie-icons-configs` package)